plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/edit.tsx

```diff
@@ -26,7 +26,6 @@ import type { BlockEditProps } from '@wordpress/blocks';
 import { previewOptions } from './preview';
 import { getActiveFilters } from './utils';
 import { Inspector } from './components/inspector';
-import { PreviewDropdown } from '../components/preview-dropdown';
```

Comment:
Let's also remove the dropdown component as it's unused now.

Reply:
thanks! removing

Reply:
addressed 666055d68a260ae5c63219fa2cd8873068800677